Apps/WebClient/src/ClientApp/src/components/report/HospitalVisitReportComponent.vue

```diff
@@ -0,0 +1,194 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component, Emit, Prop, Watch } from "vue-property-decorator";
+import { Action, Getter } from "vuex-class";
+
+import { DateWrapper } from "@/models/dateWrapper";
+import { HospitalVisit } from "@/models/encounter";
+import Report from "@/models/report";
+import ReportField from "@/models/reportField";
+import ReportFilter from "@/models/reportFilter";
+import ReportHeader from "@/models/reportHeader";
+import { ReportFormatType, TemplateType } from "@/models/reportRequest";
+import RequestResult from "@/models/requestResult";
+import User from "@/models/user";
+import container from "@/plugins/container";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import { ILogger, IReportService } from "@/services/interfaces";
+
+interface HospitalVisitRow {
+    date: string;
+    health_service: string;
+    visit_type: string;
+    location: string;
+    provider: string;
+}
+
+@Component
+export default class HospitalVisitReportComponent extends Vue {
+    @Prop() private filter!: ReportFilter;
+
+    @Action("retrieveHospitalVisits", { namespace: "encounter" })
+    retrieveEncounters!: (params: { hdid: string }) => Promise<void>;
```

Comment:
Would suggest renaming the field to `retrieveHospitalVisits` to match the action.